frontend/src/static/js/v_summary.js

```diff
@@ -516,8 +525,12 @@ window.vSummary = {
         repo.users.forEach((user) => {
           Object.keys(user.fileTypeContribution).forEach((fileType) => {
             if (!Object.prototype.hasOwnProperty.call(fileTypeColors, fileType)) {
-              fileTypeColors[fileType] = selectedColors[i];
-              i = (i + 1) % selectedColors.length;
+              if (i < selectedColors.length) {
+                fileTypeColors[fileType] = selectedColors[i];
+                i += 1;
```

Comment:
note that eslint does not allow `i++`, if not I could have make this a one liner:

`fileTypeColors[fileType] = i < selectedColors.length ? selectedColors[i++] : this.getRandomColor();